### ISSUE TITLE
Teardown for capsule tests is required to delete bogus capsules

### ISSUE THREAD

**Issue by rplevka** [SatelliteQE/robottelo#3446]
We need to either implement removal of bogus capsules or create them only with an appropriate organization assigned (not sure whether this helps) straight after we're done with capsule tests.
Otherwise they point to an already closed port which causes `Connection Refused` for many other tests down the road (e.g. Content View Publish). Such tests produce locked tasks which further prevent other tasks from other tests to finish.

**Comment by rplevka** [comment]
update: Teardown might not be sufficient in parralel test execution. The correct approach would be to assign the capsule (only) to the organization created by the test - to preserve test isolation.

**Comment by rplevka** [comment]
update: There is a 'leak' in the isolation due to the following bug:
https://bugzilla.redhat.com/show_bug.cgi?id=1336449
tl;dr:
- new UI tests apparently start the session with the Context of the organization created in the recent test.
- due to a BZ stated above newly created orgs (via UI) inherit the settings (e.g. Capsule association) from the Organization which context is currently being used.

- **expected behaviour:** The `Manage Organizations` page always resets the context to `Any Context`
- **workaround:** explicitely specify the Organization ID while creating a new proxy using `make_proxy()` factory method

**Comment by sghai** [comment]
@rplevka @sthirugn : can we close this issue  ? I see that referenced PR handled the removal of fake capsules ?

**Comment by rplevka** [comment]
@sghai, yes..thanks for reminder
